docs/state.md

```diff
@@ -103,11 +103,10 @@ The state store can easily be moved to a different s3 bucket. The steps for a si
 
 Repeat for each cluster needing to be moved.
 
-#### Cross Account State-store (AWS S3)
+#### Cross Account State-store
 
-There are situations in which the entity executing kops to create the cluster is not in the same account as the owner of the state store bucket. In this case, you must explicitly grant the permission: `s3:getBucketLocation` to the ARN that is running kops.
-
-You can use the following policy to guide your implementation:
+Many enterprises prefer to run many AWS accounts. In these setups, having a shared cross-account S3 bucket for state may make inventory and management easier.
+In order to achieve this, you first need to let the other accounts access the s3 bucket. This is done by adding the following _bucket policy_ on the S3 bucket:
```

Comment:
I find it easiest to describe multi-account scenarios by referring to them as Account A and Account B. Maybe we could do something similar here?

```suggestion
Consider the S3 bucket living in Account B and the kops cluster living in Account A. In order to achieve this, you first need to let Account A access the s3 bucket. This is done by adding the following _bucket policy_ on the S3 bucket:
```

and then update policy document to reference `arn:aws:iam::account-a:root` or something. and mention them again in the paragraph below.

Thoughts?

Reply:
Yeah, it is better. That makes it more consistent with the AWS docs too, I think. Thanks.